schema/Retail/RET11/on_init.ts

```diff
@@ -555,7 +553,7 @@ export const FnBonInitSchema = {
               },
             },
           },
-          required: ['provider', 'provider_location', 'items', 'billing', 'fulfillments', 'quote', 'payment', 'tags'],
+          required: ['provider', 'items', 'billing', 'fulfillments', 'quote', 'payment', 'tags'],
```

Comment:
additional properties can be set as false to throw errors in case provider_location and other such attributes are provided